catalyst-gateway-crates/c509-certificate/Cargo.toml

```diff
@@ -59,6 +59,17 @@ strum_macros = "0.26.3"
 regex = "1.10.5"
 ed25519-dalek = { version = "2.1.1", features = ["pem"] }
 thiserror = "1.0.56"
+serde = { version = "1.0.204", features = ["derive"] }
 
 [package.metadata.cargo-machete]
 ignored = ["strum"]
+
+[dev-dependencies]
+clap = { version = "4.5.9", features = ["derive"] }
+serde_json = "1.0.120"
+rand = "0.8.5"
+chrono = "0.4.38"
```

Comment:
What are used or included during the runtime should move to `dependencies`. I think `clap` is the obvious one.

Reply:
Resolved since all of this is used in `examples`